src/test/scala/firrtlTests/LoweringCompilersSpec.scala

```diff
@@ -311,7 +241,7 @@ class LoweringCompilersSpec extends AnyFlatSpec with Matchers {
     compare(expected, tm)
   }
 
-  it should "work for Mid -> Mid" in {
+  it should "work for Mid -> Mid" ignore {
```

Comment:
Is this worth ignoring or just deleting?

Reply:
I was mostly following convention :)

(If you look carefully, some of the other "work for ... -> ..." tests are already ignored)

In the end we need someone brave enough to just remove 98% of these LoweringCompiler tests as they are way too strict.

Reply:
If someone wants to just PR removal of `LoweringCompilersSpec`, I will immediately approve it. This has clearly outlived its welcome. 😄

Reply:
Seems like I have thought about that in the past:
![Screenshot from 2021-08-02 13-42-06](https://user-images.githubusercontent.com/2456545/127921216-957f022b-3561-464a-a9ae-c9ae96bb3343.png)
(the grey text is the auto complete suggestion)